tests/data/run_cancel_response.json

```diff
@@ -0,0 +1,59 @@
+{
```

Comment:
Did a cancel of a run and added this response json. Had also compared to some of the existing json responses and redacted or changed some id's (such as the `account_id`, `project_id`, etc) accordingly.